tools/ops/hue_collect_data.sh

```diff
@@ -267,12 +265,7 @@ main()
 
          if [[ ! -z ${COLLECT_THREADS} ]]
          then
-            echo "Getting a thread dump:"
-            ${MKDIR} `dirname ${HUE_THREADS_FILE}`
-            do_curl \
-                 GET \
-                 "${HUE_THREADS_URL}" \
-                 -L -o ${HUE_THREADS_FILE}_${DATE}
+            echo "Shifted from CherryPy server to Gunicorn server. Can't collect Threads here"
```

Comment:
why not remove everything concerning threads there then? i.e. the if statements as well

Reply:
@bjornalm 
We are actually thinking to remove the entire hue_collect_data.sh file. The final decision needs to be discussed.

Reply:
Discussed with @JohanAhlen too. We will delete the entire hue_collect_data.sh file in this PR itself.